scaleway/extra-variables.tf

```
@@ -0,0 +1,4 @@
+variable "lb_id" {
+  type        = string
+  description = "ID of the main load balancer"
```

For a question of maintaining style with the other modules, the description should come first. I also added a `nullable = false` because I think this value is required and we cannot risk anybody passing a `null` value and cause an undefined behavior (that is something I also should have done on the SKS variant and did not do it).

```suggestion
  description = "ID of the Scaleway Load Balancer to use for the Kubernetes cluster."
  type        = string
  nullable    = false
```